pkg/tsdb/cloudwatch/models/request/resource_request.go

```diff
@@ -0,0 +1,22 @@
+package request
+
+import (
+	"fmt"
+	"net/url"
+)
+
+type ResourceRequest struct {
+	Region string
```

Comment:
Currently, it's only region that is common for all resource requests, but there will be more in the future.